src/Cache/_ScsDataClient.php

```diff
@@ -288,6 +282,33 @@ public function delete(string $cacheName, string $key): CacheDeleteResponse
         return new CacheDeleteResponseSuccess();
     }
 
+    public function keysExist(string $cacheName, array $keys) : CacheKeysExistResponse
+    {
+        try {
+            validateCacheName($cacheName);
+            $keysExistRequest = new _KeysExistRequest();
+            $keysExistRequest->setCacheKeys($keys);
```

Comment:
Do we want to validate `$keys` before passing them like [this](https://github.com/momentohq/client-sdk-php/blob/ad4305cd87b8ea9bd74f962239e2f406deac0cd5/src/Utilities/_DataValidation.php#L97)?

Reply:
Great callout, I'll add that, thanks!